packages/edit-site/src/components/global-styles/variations/style.scss

```diff
@@ -5,34 +5,45 @@
 	cursor: pointer;
 
 	.edit-site-global-styles-variations_item-preview {
-		padding: $border-width * 2;
 		border-radius: $radius-block-ui;
-		box-shadow: 0 0 0 $border-width $gray-200;
 		// Shown in Windows 10 high contrast mode.
 		outline: 1px solid transparent;
+		position: relative;
+		transition: box-shadow 0.1s linear;
+
+		&::after {
```

Comment:
Should this be a mixin?

Reply:
If not then at least I think we should have a comment that explains what it does.

Reply:
We don't currently have a mixin for this. I did add a comment and the reduce-motion mixin for these: https://github.com/WordPress/gutenberg/pull/60031/commits/1fdbbeca8eb22ade40f560fcc822ca41f6ec8fa4